common/xfer/plugin_spec.go

```diff
@@ -7,8 +7,8 @@ import (
 	"sort"
 
 	"github.com/davecgh/go-spew/spew"
-	"github.com/mndrix/ps"
 	"github.com/ugorji/go/codec"
+	"github.com/weaveworks/ps"
```

Comment:
Done https://github.com/mndrix/ps/pull/5

I commit to replace the vendoring by the upstream version if the changes get merged.